scripts/install.sh

```diff
@@ -0,0 +1,25 @@
+# Install all dependencies
+# Check if it is macos runtime or ubuntu
+# Assuming this is reun inside a python3 devcontainer
+set -e # exit on error
+
+sudo apt-get update && sudo apt-get upgrade
+curl https://bootstrap.pypa.io/get-pip.py -o get-pip.py
+python get-pip.py
+curl -sSL https://install.python-poetry.org | python -
+poetry run pip install --upgrade pip setuptools toml
+if [[ "$(uname)" == "Darwin" ]]; then
+	brew install zbar
+elses
```

Comment:
I see the unit test is not installing `zbar` is this a typo? 

```suggestion
else
```

Reply:
oops!